redis_hashring/__init__.py

```diff
@@ -315,13 +324,11 @@ def poll(self):
 
         try:
             while True:
-                # Since Redis' listen method blocks, we use select to inspect the
-                # underlying socket to see if there is activity.
-                fileno = pubsub.connection._sock.fileno()
                 timeout = max(0, POLL_INTERVAL - (time.time() - last_heartbeat))
-                r, w, x = self._select([fileno], [], [], timeout)
-                if fileno in r:
-                    next(gen)
```

Comment:
Above, is `gen = pubsub.listen()` still needed?

Reply:
Dropped it in https://github.com/closeio/redis-hashring/pull/3/commits/596f4aa1ef34e8f6b34b1624aec4f5b453771142.